backend/dataall/db/api/pipeline.py

```diff
@@ -295,6 +295,40 @@ def delete_pipeline_environments(session, uri) -> bool:
         session.commit()
         return True
 
+    @staticmethod
+    def delete_pipeline_environment(
```

Comment:
I think this method can be implemented in an easier way by calling the pipelineEnvironment unique identifier directly. "envPipelineUri" This would avoid filtering by pipelineUri and environmentUri and stage. What do you think?

Reply:
I agree would make the look up for delete simpler - pushed new commit only relying on unique identifier of envPipelineUri to find and delete the pipeline development environment